package.json

```diff
@@ -37,7 +37,7 @@
     "@material-ui/core": "4",
     "@material-ui/lab": "^4.0.0-alpha.57",
     "@testing-library/react": "^11.2.2",
-    "cozy-client": "22.1.0",
+    "cozy-client": "23.10.0",
```

Comment:
this commit should be part of the previous fix. Since the previous one will not work without this upgrade of cozy-client. They should be "linked"